uiTests/src/test/resources/project.rhino.js

```diff
@@ -24,11 +24,15 @@ function Project(underlyingProject) {
   };
 
   // Tab & model management
-  this.openTab = function () {
+  this.openGitMacheteTab = function () {
+    this.openTab(ToolWindowId.VCS, 'Git Machete');
+  }
+
+  this.openTab = function (toolWindowId, tabName) {
```

Comment:
Nit: I guess some of these method could be "private" (`const foo = function...`)